src/Silhouette.js

```diff
@@ -106,7 +108,7 @@ class Silhouette {
             }
             ctx.clearRect(0, 0, width, height);
             ctx.drawImage(bitmapData, 0, 0, width, height);
-            imageData = ctx.getImageData(0, 0, width, height);
+            imageData = safeGetImageData(ctx, width, height);
```

Comment:
Unnecessary here because we check and return early if the passed `width` or `height` are 0

Reply:
I think since it's not too much extra operation, it's a good idea to have a standard to use it everywhere, for safety